### PR TITLE
Improve sorting and displayed age in task lists

### PR DESCRIPTION
**Pull request by vasilty**

### CODE REVIEW

File: rt_dashboard/web.py [hunk 1]
```diff
@@ -257,7 +257,9 @@ def list_jobs(queue_name, page):
         )
 
         offset = (current_page - 1) * per_page
-        jobs = [serialize_job(job) for job in queue.get_tasks(offset, per_page)]
+        jobs = sorted(
+            (serialize_job(job) for job in queue.get_tasks(offset, per_page)),
+            key=itemgetter('enqueued_at'), reverse=True)
```
Comment (Chronial) [inline rt_dashboard/web.py]: You realize that this only changes the sorting on the current page, right? But it is wrong at the moment?

Comment (vasilty) [reply rt_dashboard/web.py]: Right, we need to actually get_tasks() in the right order

**Review by Chronial — APPROVED** [review]
Does this fix the date issue? If yes, this LGTM.

**Comment by Chronial** [comment]
Integrated into #4.

**Pull request closed without merging**